pages/nft-book-store/index.vue

```diff
@@ -266,11 +265,6 @@ const tableColumns = [
     key: 'sold',
     label: 'Sold',
     sortable: true
-  },
-  {
-    key: 'stock',
-    label: 'Remaining Stock',
-    sortable: true
   }
 ]
```

Comment:
Why remove this?

Reply:
Daisy was confused by the terms _remaining stock_, _unassigned stock_, and _stock_. Since _remaining stock_ becomes less meaningful when there is more than one edition, I followed Daisy’s suggestion to remove it and instead display the _stock_ of each edition on the /status/class page.